### PR TITLE
Add default whistle + whistles reorganize

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

Move all whistle sprites into one folder
Add grey whistle in arcade
Add equipped sprites for all whistles

## Why / Balance

Just feels good to give crew the whistle that won't get them arrested

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Grey whistle added to arcade machine prizes.

### CODE REVIEW

**Review by VasilisThePikachu — DISMISSED** [review]
Resolve test fail
```
A total of 1 test files matched the specified pattern.
  Skipped TestEmptyLoadout [< 1 ms]
  Failed AllItemsHaveSpritesTest [1 s]
  Error Message:
   Multiple failures or warnings in test:
  1) Item prototype has no sprite: Whistle. It should probably either be marked as abstract, not be an item, or have a valid sprite
  2) Item prototype has no sprite: SyndicateWhistle. It should probably either be marked as abstract, not be an item, or have a valid sprite
  3) Item prototype has no sprite: SecurityWhistle. It should probably either be marked as abstract, not be an item, or have a valid sprite
  ```

**Comment by lzk228** [comment]
i dont understand what is the problem
the path is written in the common parent and all icon states is written in that prototypes

File: Resources/Prototypes/Entities/Objects/Fun/whistles.yml [hunk 1]
```diff
@@ -16,18 +20,26 @@
...
   - type: Whistle
-    distance: 5
+    distance: 3
+
+- type: entity
+  parent: BaseWhistle
+  id: Whistle
+  components:
+  - type: Sprite
+    icon: whistle
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Objects/Fun/whistles.yml:30]:
```suggestion
    state: whistle
```

File: Resources/Prototypes/Entities/Objects/Fun/whistles.yml [hunk 2]
```diff
@@ -16,18 +20,26 @@
...
   parent: BaseWhistle
   id: SecurityWhistle
   description: Sound of it make you feel fear.
   components:
   - type: Sprite
-    sprite: Objects/Fun/whistle.rsi
-    state: securityWhistle
-  - type: Item
-    sprite: Objects/Fun/whistle.rsi
+    icon: sec
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Objects/Fun/whistles.yml:38]:
```suggestion
    state: sec
```

File: Resources/Prototypes/Entities/Objects/Fun/whistles.yml [hunk 3]
```diff
@@ -36,13 +48,9 @@
   description: A whistle used by Syndicate commanders to draw attention. Avanti!
   components:
   - type: Sprite
-    sprite: Clothing/Neck/Misc/whistles.rsi
-    state: icon
+    icon: trench
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Objects/Fun/whistles.yml:51]:
```suggestion
    state: trench
```

**Review by Tayrtahn — COMMENTED** [review]
?

**Pull request merged**